discover/deployment/articles-dxp/05-maintaining-liferay/01-patching-liferay-dxp/03-configuring-the-patching-tool/03-installing-patches-on-the-liferay-war.markdown

```diff
@@ -46,36 +46,34 @@ Download the necessary artifacts from the
         module.framework.portal.path=/osgi-home/osgi/portal
         module.framework.static.path=/osgi-home/osgi/static	
 
-3.  Download the patch (fix pack or hotfix) to install and put it in the
-    Patching Tool's `patches` folder. 
+3.  Download the patch (fix pack or hotfix) to install and put it in a folder 
+    called `/patching-home/patches`. 
```

Comment:
The patches folder is inside the patching-tool folder so this path is not correct, could you please fix it?

Reply:
Good catch!